store/templates/navbar.html

```diff
@@ -0,0 +1,34 @@
+<!-- Navigation-->
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="container px-4 px-lg-5">
+        <a class="navbar-brand" href="{% url 'store:home' %}">Django Ecommerce</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"><span
+                class="navbar-toggler-icon"></span></button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0 ms-lg-4">
+                <li class="nav-item"><a class="nav-link active" aria-current="page" href="{% url 'store:home' %}">Home</a></li>
+                <li class="nav-item"><a class="nav-link" href="{% url 'store:about' %}">About</a></li>
```

Comment:
This took me a bit of time to figure out why it was raising an exception. Which was due to missing `store:` prefix as I've defined it in the `urls.py` file